src/dot/commons/video/video_utils.py

```diff
@@ -23,7 +83,7 @@ def video_pipeline(
     change_option: Callable[[np.ndarray], None],
     process_image: Callable[[np.ndarray], np.ndarray],
     post_process_image: Callable[[np.ndarray], np.ndarray],
-    crop_size: int = 224,
+    crop_size: int,
```

Comment:
will this break backwards compatibility? it might as you are changing this variable from optional to mandatory.